python/cudf/cudf/_lib/pylibcudf/libcudf/io/types.pxd

```diff
@@ -81,6 +81,7 @@ cdef extern from "cudf/io/types.hpp" \
         map[string, string] user_data
         vector[unordered_map[string, string]] per_file_user_data
         vector[column_name_info] schema_info
+        vector[size_t] num_rows_per_source
```

Comment:
Added as per @galipremsagar's suggestion. No need to do anything else with it as of now!

Reply:
Just put here for data structure completeness. Currently being ignored in `.pyx` and `.py` files so nothing changes at Python side.